kpatch-build/create-diff-object.c

```diff
@@ -1213,8 +1253,44 @@ static void kpatch_replace_sections_syms(struct kpatch_elf *kelf)
 
 				rela->sym = sym;
 				rela->addend -= start;
+				found = 1;
 				break;
 			}
+			/*
+			 * Unbundled sections might refer to SHN_UNDEF symbol type, example kernel/sched/core.o.
+			 * Relocation section '.rela.data' at offset 0xadda8 contains 2 entries:
+			 *    Offset             Info             Type       Symbol's Value   Symbol's Name + Addend
+			 * 0000000000000000  000003ae00000026 R_PPC64_ADDR64 0000000000000000 call_rcu_sched + 0
+			 * 0000000000000008  000003ae00000026 R_PPC64_ADDR64 0000000000000000 call_rcu_sched + 0
+			 *
+			 * Replace them with reference to global symbols.
+			 */
+			if (!found) {
+				struct section *rela_sec;
+				struct rela *rela1;
+
+				if (!rela->sym->sec->rela)
+					continue;
+
+				rela_sec = find_section_by_name(&kelf->sections, rela->sym->sec->rela->name);
```

Comment:
I think this is unnecessary, it can just be
```
rela_sec = rela->sym->sec->rela;
```

Reply:
Agree, it can be made simpler by removing `rela_sec`
```
if (!found) {
    struct rela *rela1;
    
    if (!rela->sym->sec->rela)
        continue;

    list_for_each_entry(rela1, &rela->sym->sec->rela->relas, list) {
```